migrations/tenant/0028-objects-level-index.sql

```diff
@@ -0,0 +1,3 @@
+-- postgres-migrations disable-transaction
+CREATE INDEX CONCURRENTLY IF NOT EXISTS "objects_bucket_id_level_idx"
```

Comment:
Do you not create an UNIQUE index here because you want to keep existing UNIQUE index?